game/i18n.cc

```diff
@@ -120,19 +125,24 @@ std::string TranslationEngine::getLanguageByKey(const std::string& languageKey)
 	return allLanguages.at(languageKey);
 }
 
-const std::pair<std::string, std::string>& TranslationEngine::getCurrentLanguage() const { 
-	return m_currentLanguage; 
+std::pair<std::string, std::string> const& TranslationEngine::getCurrentLanguage() { 
+	return TranslationEngine::m_currentLanguage; 
+}
+
+std::string TranslationEngine::getCurrentLanguageCode() {
+	std::string lang(getCurrentLanguage().first);
+	return lang.substr(0, lang.size() - 6);
```

Comment:
Whats this minus 6??

Reply:
Sorry for the magic number, it's ".UTF-8"

Reply:
Maybe it'd be best to calculate the cut-off in relation to the index of the first `.`?